test/models/playlist_test.rb

```diff
@@ -1,44 +1,74 @@
 require "test_helper"
 
 class PlaylistTest < ActiveSupport::TestCase
-  test "invalid if name is blank" do
-    playlist = Playlist.new(name: "", energy: 35, danceability: 56, genre: "genre1")
-    playlist.valid?
-    assert_not playlist.errors[:name].empty?
+
+  def valid_playlist_params
+    {
+      name: 'name1',
+      energy: 35,
+      danceability: 56,
+      genre: "genre1"
+    }
   end
 
-  test "invalid if all parameters are blank" do
-    playlist = Playlist.new(name: "name1")
+  test "should validate name presence" do
+    playlist = Playlist.new(valid_playlist_params.merge(name: ''))
+    assert_not playlist.valid?
+  end
+
+  test "should validate that all three parameters cannot be blank" do
+    playlist = Playlist.new(valid_playlist_params.merge(energy: nil, danceability: nil, genre: ""))
     playlist.valid?
     assert_not playlist.errors[:base].empty?
```

Comment:
```suggestion
    assert_not playlist.valid?
```

comme précisé dans un commentaire ultérieur que j'ai fait sur le première test

Reply:
Oups désolée j'avais compris que c'était valable seulement pour le premier test